test/resources/language/typescript/tslint.json

```diff
@@ -0,0 +1,94 @@
+{
```

Comment:
Would mind to strip this configuration file down to the bare minimum?  It's rather long currently, and for us who don't know typescript it's very hard to understand which of these rules are actually required for the test case and must be edited with care.